src/routes.ts

```diff
@@ -83,15 +85,15 @@ const navOptScreenWCancel = {
 
 export const BottomTabBarRoutes = {
   [routeList.Claims]: {
-    screen: Claims,
+    screen: withNotification(withSticky(Claims)),
```

Comment:
does this work as expected? doesn't it render notifications twice?

Reply:
It doesn't. `withNotifications` renders all notifications except the sticky ones.